resources/factions/japan_2005.json

```diff
@@ -9,8 +9,9 @@
   "aircrafts": [
     "AH-1W SuperCobra",
     "AH-64D Apache Longbow",
+    "C-130",
+    "C-130J-30 Super Hercules",
     "F-15J Eagle",
-    "F-2A",
```

Comment:
Not sure why this was deleted? I'm adding it back because it looks unintentional.